src/Makefile.am

```diff
@@ -7,6 +7,7 @@ SHARED_EXT=@SHARED_EXT@
 pkcs11_provider_la_SOURCES = \
 	asymmetric_cipher.c \
 	debug.c \
+	endian.h \
```

Comment:
think this need to be changed accordingly

Reply:
Yep, good catch, fixed.